src/Server/WebServer.fs

```diff
@@ -16,13 +17,16 @@ let webApp databaseType root =
 
     router notfound [
         GET [
-            route "/" (htmlFile (System.IO.Path.Combine(root,"index.html")))
```

Comment:
I assum index.html can now be deleted completely!?

Reply:
Not really, since we still use the client router, we can customize 404 pages on the client side.